htdocs/main.php

```diff
@@ -80,12 +80,16 @@ function tplFromRequest($param)
 // draw the user information table
 try {
     $user =& User::singleton();
+    $site_arr = $user->getData('CentersID');
+    foreach ($site_arr as $key=>$val) {
+        $site[$key] = & Site::singleton($val);
+        $isStudySite[$key] = $site[$key]->isStudySite();
+    }
+    $oneIsStudySite = in_array("1", $isStudySite);
```

Comment:
what would happen if the user is in multiple sites, and some are StudySites while others are not? I was not sure in this case what to return (true or false), so I thought if at least one is StudySite then return true...